Packs/MISP/Integrations/MISPV3/README.md

```diff
@@ -3934,27 +3934,27 @@ Search for attributes in MISP.
+| org | Search by the creator organization by supplying the organization identifier.                                                                                                                                                                      | Optional | 
+| tags | A comma-separated list of tags to include in the results. To exclude a tag, prefix the tag name with "!". Can be: "AND", "OR", and "NOT" followed by ":". To chain logical operators use ";". For example, "AND:tag1,tag2;OR:tag3".               | Optional | 
+| from | Events with the date set to a date after the one specified. This filter will use the date of the event.                                                                                                                                           | Optional | 
+| to | Events with the date set to a date before the one specified. This filter will use the date of the event.                                                                                                                                          | Optional | 
+| event_id | A comma-separated list of event IDs. Returns the attributes that are part of the given event IDs.                                                                                                                                                 | Optional | 
+| include_sightings | Whether to include the the sightings of the matching attributes. Default is false. Possible values: "true" and "false". Possible values are: true, false.                                                                                         | Optional | 
+| include_correlations | Whether to include the full correlations of the matching attributes. Possible values: "true" and "false". Default is false. Possible values are: true, false.                                                                                     | Optional | 
+| page | If a limit is set, sets the page to be returned. For example, page 3, limit 100 will return records 201-&gt;300. Default is 1. Default is 1.                                                                                                      | Optional | 
+| limit | Limit the number of attributes returned. Default is 50. Default is 50.                                                                                                                                                                            | Optional | 
+| enforceWarninglist | Whether to return only the values that are not on the warninglists. Possible values: "true" and "false". Possible values are: true, false.                                                                                                        | Optional | 
+| compact | Whether to return only the attribute's values that match the search query. In case you want to get the full attributes data, set this argument to false. Possible values: "true" and "false". Possible values are: true, false. Default is false. | Optional |
```

Comment:
```suggestion
| compact | Whether to return only the attribute's values that match the search query. In case you want to get the full attributes data, set this argument to false. Possible values are: true, false. Default is false. | Optional |
```